ngraph/core/builder/src/builder/autobroadcast.cpp

```diff
@@ -447,6 +437,25 @@ namespace ngraph
                 return op::Constant::create(element::i64, Shape{axes_mapping.size()}, axes_mapping);
             }
 
+            Output<Node> get_axes_mapping_output(const PartialShape& output_shape,
+                                                 const Output<Node>& input_shape,
+                                                 std::size_t start_match_axis)
+            {
+                const auto one_node = opset7::Constant::create(element::i64, Shape{}, {1});
```

Comment:
Any particular reson to use opset7 directly instead of just `op`? The same question goes to the ops referenced below.

Reply:
it's easier to update operators versions to the newest opsets this way.